src/components/camera/camera.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {
+  Container,
+  Info,
+  Title,
+  SubTitle,
+  ScanTitle,
+  CameraContainer,
+  Input,
+  Button,
+} from "./camera.styles";
+
+/**
+ * Camera Component
+ * @author [Colburn Sanders](https://github.com/colburncodes)
+ */
+
+export const Camera = ({ title, caption, subTitle, inputText, buttonText }) => {
+  return (
+    <>
+      <Container className="d-flex text-center bg-alpha">
+        <Info className="flex-column">
+          <Title>{title}</Title>
+          <SubTitle>{caption}</SubTitle>
+          <ScanTitle>{subTitle}</ScanTitle>
+          <CameraContainer className="flex-column"></CameraContainer>
```

Comment:
I think let's remove the items from this component that aren't strictly related to the camera-- this should essentially only contain
```
<CameraContainer className="flex-column"></CameraContainer>
```

The thinking is that there may, at some point in the future, be cases where we want to re-use a camera component somewhere on the page but it may not have the input text or button beneath it, or any of the title/subtitle/caption info, so those shouldn't be strongly coupled to this component in that way. Those elements belong in the layout of the camera step when we eventually build that; for now, if you don't want to waste those styles, you can create a directory in `routes`, such as `routes/add-account` and you can put most of these styled components in there since that's most likely where we will eventually use them. If you choose to do that, put a comment in the `index.js` file of that directory that explains what it is going to be used for